src/components/Favorites/Favorites.style.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+export const FavoritesTitle = styled.h1`
```

Comment:
I am confused a little bit, why this lives below `../Favorites/` instead of having a style for audio player, Im not 100% sure of how we separate these styles now.

cc @tonymtz